go/galeraautofix/mariadb/mariadb.go

```diff
@@ -188,6 +188,68 @@ func IsDBAvailable(ctx context.Context, host string) bool {
 	return false
 }
 
+func IsLocalDBReady(ctx context.Context) bool {
+	return IsDBReady(ctx, "localhost")
+}
+
+func IsDBReady(ctx context.Context, host string) bool {
```

Comment:
Just to make sure, these are added here but the actual galera-autofix code is not being altered?

Reply:
It won't change anything, i just want to rely on the great job that has been done in galera-autofix.